src/ucs/debug/log.c

```diff
@@ -145,15 +145,24 @@ static void ucs_log_get_file_name(char *log_file_name, size_t max, int idx)
 
 static void ucs_log_file_rotate()
 {
-    char old_log_file_name[PATH_MAX];
-    char new_log_file_name[PATH_MAX];
+    char *old_log_file_name = NULL;
+    char *new_log_file_name = NULL;
     int idx, ret;
 
+    old_log_file_name = (char *)ucs_malloc(PATH_MAX, "old_log_file_name");
+    new_log_file_name = (char *)ucs_malloc(PATH_MAX, "new_log_file_name");
+
+    if (old_log_file_name == NULL || new_log_file_name == NULL) {
+        ucs_free(old_log_file_name);
```

Comment:
I guess no need to free when we `ucs_fatal()`.

Reply:
valid for all cases.

Reply:
Thank you :), please check now @tvegas1

Reply:
Outdated, done in other PRs